bot/exts/evergreen/snakes/_converter.py

```diff
@@ -1,7 +1,7 @@
 import json
 import logging
 import random
-from typing import Iterable, List
+from typing import Iterable
```

Comment:
According to PEP 585, just like with `List`, `Dict`, etc. importing this kind of type from `typing` is also deprecated; `collections.abc` should be used. You should refer to that PEP for the full list.

Reply:
Oh dear god, you are correct. That's a big list. I'll implement that tomorrow

Reply:
_Narrator: it wasn't implemented tomorrow._
I'll allocate some time later, I suppose this is not a very high-priority task